server/service/src/sync/translations/invoice.rs

```diff
@@ -176,21 +173,20 @@ pub struct LegacyTransactRow {
     /// time in seconds
     #[serde(deserialize_with = "naive_time")]
     pub entry_time: NaiveTime, // e.g. 47046,
-    /// shipped_datetime
-    #[serde(deserialize_with = "zero_date_as_option")]
-    #[serde(serialize_with = "date_option_to_isostring")]
-    pub ship_date: Option<NaiveDate>, // "0000-00-00",
```

Comment:
Not sure why we're populating ship_date and arrival_date_actual with shipped and delivered... we generate these times and I'm guessing actual shipping time and arrival times will differ and we don't have that in OMS UI anyway...